web/utils.py

```
@@ -273,7 +275,9 @@ def get_leaderboard(current_user=None, period=None, limit=10):
     elif period == "monthly":
         # Get monthly leaderboard
         leaderboard_entries = (
-            Points.objects.filter(awarded_at__gte=one_month_ago, user__profile__is_teacher=False)
+            Points.objects.filter(
+                awarded_at__gte=one_month_ago, user__profile__is_teacher=False, user__profile__is_profile_public=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)